__tests__/components/ExportURLInput.test.tsx

```diff
@@ -1,11 +1,18 @@
 import { render, screen, fireEvent } from '@testing-library/react';
 import '@testing-library/jest-dom';
 import { mantineWrap } from '../helpers/testHelpers';
+import { RecoilRoot } from 'recoil';
 import ExportURLInput from '../../components/ExportURLInput';
 
 describe('ExportURLInput', () => {
   it('renders a text input field', () => {
-    render(mantineWrap(<ExportURLInput />));
+    render(
+      mantineWrap(
+        <RecoilRoot>
```

Comment:
We can/should move the RecoilRoot to the mantineWrap setup function itself, since every component unit test will likely need it? 

Maybe we can rename that function to just a general `providerWrap` or something since we might have other things that wrap the whole app